pack/Microsoft.NET.Sdk.Functions/Microsoft.NET.Sdk.Functions.csproj

```diff
@@ -0,0 +1,61 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
+    <PackageName>Microsoft.NET.Sdk.Functions</PackageName>
+    <PackageVersion>1.0.3</PackageVersion>
+    <AssemblyOriginatorKeyFile>PublicKey.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>False</SignAssembly>
+    <DelaySign>True</DelaySign>
+    <BuildOutputTargetFolder>tools</BuildOutputTargetFolder>
+    <FunctionsOutputPath>..\..\src\Microsoft.NET.Sdk.Functions.MSBuild\bin\Release</FunctionsOutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="..\..\src\Microsoft.NET.Sdk.Functions.MSBuild\Targets\netstandard1.0\**\*">
+      <Pack>true</Pack>
+      <PackagePath>build\netstandard1.0\</PackagePath>
+    </Content>
+    <Content Include="..\..\src\Microsoft.NET.Sdk.Functions.MSBuild\Targets\Sdk\**\*">
+      <Pack>true</Pack>
+      <PackagePath>Sdk\</PackagePath>
+    </Content>
+  </ItemGroup>
+
+  <Target Name="PackReferenceAssemblies" AfterTargets="Build">
+    <ItemGroup>
+      <None Include="$(FunctionsOutputPath)\net46\Newtonsoft.Json.dll">
```

Comment:
Other than Newtonsoft.Json, do you expect any other dlls right next to the msbuild task dll?

Reply:
@ahmelsayed

Reply:
I thought we used to have System.ValueTuple as well, though I guess it can be removed by not using the new tuples.

Reply:
nevermind, you're right. I used to use value tuples in the reflection build task itself but that code was removed a while ago.

Btw, I just noticed I never merged my PR #109, should I just close that in favor of this or should I merge it still?

Reply:
You can close the other PR. This should handle 
- Task assembly targeting netstandard1.5 
- & package & the dependencies targeting netstandard2.0

Reply:
awesome!